### ISSUE TITLE
PCL profile cleanup

### ISSUE THREAD

**Issue by hvanbakel** [hvanbakel/CsprojToVs2017#197]
Split off from #165 

PCL profiles to .NET Standard conversion can be 2.x scheduled. There is about 12 supported PCL profiles (of approximately 80), but they are the most common.

```xml
<Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
```
The one above can be made autoremoved in 2.x if no PCL profiles are left in TargetFrameworks.

**Comment by hvanbakel** [comment]
@andrew-boyarshin so this would mean rewriting the profiles mentioned [here](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#comparison-to-portable-class-libraries) to the corresponding net standard version?

In the case of a non supported one we should abort, right?

**Comment by andrew-boyarshin** [comment]
Yeah, that seems like it. Although [this page](https://portablelibraryprofiles.stephencleary.com) is better in that respect (don't forget to enable showing legacy profiles).
